templates/master/00-master/baremetal/files/baremetal-clusterinfo.yaml

```diff
@@ -0,0 +1,18 @@
+filesystem: "root"
+mode: 0755
+path: "/usr/local/bin/clusterinfo"
```

Comment:
We should probably ship stuff like this with RHCOS generically across platforms.

Reply:
Any pointers on doing so would be appreciated.

Reply:
@cgwalters I have the same needs, I need DNS_VIP, API_VIP, DOMAIN and so on and I'm suggesting to add the `install-config.yaml` values to RenderConfig and by that we would have access to all the data coming from the installer, during render time. All places in the code will become just:
```
{{ .Domain }}
{{ (split "." .Domain -1) | first }}   // for the host
{{ .Platform.SomeSurveryValue }}
```